### PR TITLE
Renaming buildtest help to buildtest show

### PR DESCRIPTION
**Pull request by Mendi03**
Made the following updates:

- Switched the functionality of `buildtest help` to `buildtest show` in order to create new purpose for `buildtest help` command.
- Updated documentation
- **_test_help.py_** was changed to **_test_show.py_**
- Instances of `buildtest help` were changed to `buildtest show` in the **_show.py_** and _**test_show.py**_ file
- **_print_help_** methods changed to **_print_show_**

### CODE REVIEW

**Review by kingxleyy — COMMENTED** [review]
everything looks good to me

File: tests/cli/test_help.py
Comment (Mendi03) [inline tests/cli/test_help.py:1]: @shahzebsiddiqui should I change the name of the function `test_buildtest_help` to `test_buildtest_show`? Furthermore, would this mean I have to change `buildtest_help` to `buildtest_show`?

Comment (shahzebsiddiqui) [reply tests/cli/test_help.py:1]: yes please change the name of method

**Review by shahzebsiddiqui — APPROVED** [review]
LGTM. Great job @Mendi03

**Pull request merged**